pkg/base1/test-file.ts

```diff
@@ -254,10 +257,13 @@ QUnit.test("modify with conflict", async assert => {
             })
             .finally(() => assert.equal(n, 1, "callback called once"))
 
+            // @ts-expect-error cannot represent modify promise signature with tag
             .then(async (content, tag) => {
                 let n = 0;
                 await cockpit.spawn(["bash", "-c", "sleep 1; echo XYZ > " + dir + "/baz"]);
                 await file.modify(old => {
+                    if (!old)
```

Comment:
We have cockpit.assert() for this.

Reply:
Thanks! Moved all of them to cockpit.assert().